build/base/Dockerfile

```diff
@@ -6,22 +6,29 @@ FROM debian:buster-slim
 # |
 # | Libraries used in the build process of this image.
 # |
+RUN apt-get update &&\
+    apt-get --no-install-recommends install -y ca-certificates curl gnupg procps &&\
+    rm -rf /var/lib/apt/lists/*
 
-RUN echo "deb http://httpredir.debian.org/debian/ buster main contrib non-free" > /etc/apt/sources.list &&\
-    apt-get update &&\
-    apt-get install -y curl wget gnupg ttf-mscorefonts-installer procps
+# |--------------------------------------------------------------------------
+# | Microsoft font installer
+# |--------------------------------------------------------------------------
+RUN apt-get update &&\
+    curl -o ./ttf-mscorefonts-installer_3.8_all.deb http://httpredir.debian.org/debian/pool/contrib/m/msttcorefonts/ttf-mscorefonts-installer_3.8_all.deb &&\
```

Comment:
Is it safe to not get the deb via https?

Reply:
Looks like a typo, maybe it redirects to https?